dev/prefixer-plugin.js

```diff
@@ -0,0 +1,57 @@
+import webpack from 'webpack';
+import {escapeRegExp} from '../src/utils/regex.js';
+
+const {Compilation, sources} = webpack;
```

Comment:
we can't `import {Compilation, sources} from 'webpack'`?

Reply:
```SyntaxError: Named export 'Compilation' not found. The requested module 'webpack' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'webpack';
const {Compilation, sources} = pkg;```